proposals/flexible-vectors/FlexibleVectors.md

```diff
@@ -159,6 +159,8 @@ instructions. For the `i8` and `i16` lanes, the high bits of `x` are ignored.
 * `vec.i16.lshl(a: vec.i16, x: i32) -> vec.i16`
 * `vec.i32.lshl(a: vec.i32, x: i32) -> vec.i32`
 * `vec.i64.lshl(a: vec.i64, x: i32) -> vec.i64`
+* `vec.f32.lshl(a: vec.f32, x: i32) -> vec.f32`
+* `vec.f64.lshl(a: vec.f64, x: i32) -> vec.f64`
 
 Returns a new vector with lanes selected from the lanes of the two input
 vectors `a` and `b` by shifting lanes of the original to the left by the amount
```

Comment:
This description and pseudo code doesn't look correct, where is vector `b`?

Reply:
There were two different approaches, I guess they haven't merged correctly.